mlte/frontend/nuxt-app/pages/index.vue

```diff
@@ -305,11 +318,12 @@ async function selectModel(modelName: string, initialPageLoad: boolean) {
     selectedVersion.value = "";
   }
   if (modelName === "") {
+    versionOptions.value = [];
     clearArtifacts();
     return;
   }
 
-  await useFetch("http://localhost:8080/api/model/" + modelName + "/version", {
+  await $fetch("http://localhost:8080/api/model/" + modelName + "/version", {
```

Comment:
Could you reduce the duplication of the hardcoded localhost:8080 path in some way? Such as by having a base fetch function that fetchArtifact and these other fetches call?